ml-agents/mlagents/trainers/policy/tf_policy.py

```diff
@@ -152,6 +152,8 @@ def create_tf_graph(self) -> None:
         # We do an initialize to make the Policy usable out of the box. If an optimizer is needed,
         # it will re-load the full graph
         self.initialize()
+        # Create assignment ops for Ghost Trainer
```

Comment:
A bit strange to have Ghost specific code in the TF policy. Isn't there another reason to call init_load_weights here ?

Reply:
Previously, this was tf specific code in the ghost....

I don't need to call it ghost specific but the ghost trainer is the only thing that uses the load/get methods

Reply:
Ok, I think we can do without that comment but I don't mind leaving it either.